tests/terraform/parser/resources/plan_tags/tfplan.json

```diff
@@ -0,0 +1,4462 @@
+          "outpost_arn": true,
+          "password_data": true,
+          "placement_group": true,
+          "primary_network_interface_id": true,
+          "private_dns": true,
+          "private_ip": true,
+          "public_dns": true,
+          "public_ip": true,
+          "root_block_device": true,
+          "secondary_private_ips": true,
+          "security_groups": true,
+          "subnet_id": true,
+          "tags": {},
+          "tenancy": true,
+          "volume_tags": true,
+          "vpc_security_group_ids": true
+        }
+      }
+    },
+    {
+      "address": "aws_internet_gateway.web_igw",
+      "mode": "managed",
+      "type": "aws_internet_gateway",
+      "name": "web_igw",
+      "provider_name": "aws",
+      "change": {
+        "actions": [
+          "create"
+        ],
+        "before": null,
+        "after": {
+          "tags": {
+            "Name": "123456789123-acme-corp-igw"
+          }
+        },
+        "after_unknown": {
+          "arn": true,
+          "id": true,
+          "owner_id": true,
+          "tags": {},
+          "vpc_id": true
+        }
+      }
+    },
+    {
+      "address": "aws_kms_alias.logs_key_alias",
+      "mode": "managed",
+      "type": "aws_kms_alias",
+      "name": "logs_key_alias",
+      "provider_name": "aws",
+      "change": {
+        "actions": [
+          "create"
+        ],
+        "before": null,
+        "after": {
+          "name": "alias/123456789123-acme-corp-logs-bucket-key",
+          "name_prefix": null
+        },
+        "after_unknown": {
+          "arn": true,
+          "id": true,
+          "target_key_arn": true,
+          "target_key_id": true
+        }
+      }
+    },
+    {
+      "address": "aws_kms_key.logs_key",
+      "mode": "managed",
+      "type": "aws_kms_key",
+      "name": "logs_key",
+      "provider_name": "aws",
+      "change": {
+        "actions": [
+          "create"
+        ],
+        "before": null,
+        "after": {
+          "customer_master_key_spec": "SYMMETRIC_DEFAULT",
+          "deletion_window_in_days": 7,
+          "description": "123456789123-acme-corp-logs bucket key",
+          "enable_key_rotation": false,
+          "is_enabled": true,
+          "key_usage": "ENCRYPT_DECRYPT",
+          "tags": null
+        },
+        "after_unknown": {
+          "arn": true,
+          "id": true,
+          "key_id": true,
+          "policy": true
+        }
+      }
+    },
+    {
+      "address": "aws_lambda_function.analysis_lambda",
+      "mode": "managed",
+      "type": "aws_lambda_function",
+      "name": "analysis_lambda",
+      "provider_name": "aws",
+      "change": {
+        "actions": [
+          "create"
+        ],
+        "before": null,
+        "after": {
+          "dead_letter_config": [],
+          "description": null,
+          "environment": [
+            {
+              "variables": {
+                "access_key": "AKIAIOSFODNN7EXAMPLE",
```

Comment:
add file to : https://github.com/bridgecrewio/checkov/blob/master/.github/exclude-patterns.txt